python/cudf/cudf/core/multiindex.py

```diff
@@ -160,20 +147,20 @@ def __init__(
         self._validate_levels_and_codes(self._levels, self._codes)
 
         source_data = cudf.DataFrame()
-        for i, name in enumerate(self._codes.columns):
-            codes = as_index(self._codes[name]._column)
-            if -1 in self._codes[name].values:
+        for i, n in enumerate(self._codes.columns):
```

Comment:
Is this change just to future-proof us for when we need to use the `name` parameter by not shadowing it here?

Reply:
Yup